makefiles/Mf-embed.in

```diff
@@ -0,0 +1,39 @@
+# Mf-embed.in
+# Copyright 1984-2016 chaoseternal@gmail.com
```

Comment:
I'm a little confused about this copyright line.  It implies that you are the original author of everything in the file, and that you wrote different parts of it at various times between 1984 and 2016.  If that's actually the case then please ignore the rest of this comment.

If you are the sole author of everything in this file and you wrote it specifically for this commit then it should be `Copyright 2016 <your name>`.  (I have no idea what the legal implications of using your email address instead of a name are, but that's up to you.)

If you based this file on, say, `Mf-install.in` and made some edits then there are a few different ways to indicate that.  The one that's been recommended to me is to have multiple copyright lines like so:
`Portions Copyright 2016 <your name>`
`Portions Copyright 1984-2016 Cisco Systems, Inc.`

I'm sure one of the committers will chime in if they have a different preferred format.